src/components/Vega/EntropyPlot.js

```diff
@@ -56,6 +93,29 @@ const EntropyPlot = observer(({ width }) => {
     }
   };
 
+  const filterMap = (map, keyToRemove) => {
+    // Used to turn proteinMap and geneMap into an array of objects
+    // that can be used with the domain plot
+    const rowTwoNames = ['nsp12 - RdRp', 'orf9c', 'E', 'orf8'];
+    return Object.keys(map)
+      .filter((key) => key !== keyToRemove)
+      .reduce((arr, key, index) => {
+        map[key]['ranges'] = map[key]['segments'];
+        if (map[key]['ranges'].length > 1) {
+          // Collapse ORF1ab ranges
+          map[key]['ranges'][0][1] = map[key]['ranges'][1][1];
+          map[key]['ranges'].slice(1, 1);
+        }
+        map[key]['row'] = index % 2 === 0 ? 0 : 1;
+        if (rowTwoNames.includes(key)) {
+          map[key]['row'] = 2;
+        }
+        map[key]['abbr'] = nameToAbbr[map[key]['name']];
+        arr.push(map[key]);
+        return arr;
+      }, []);
+  };
+
```

Comment:
I'm ok with this strategy of placing the tracks manually for now, but let's open an issue to write an algorithm that does this automatically. It will be useful for #350 as well

Reply:
Opened #352